test/Faker/Provider/id_ID/TextTest.php

```diff
@@ -0,0 +1,54 @@
+<?php
+namespace Faker\Test\Provider\id_ID;
+
+use Faker\Provider\id_ID\Text;
+use Faker\Generator;
+
+class TextTest extends \PHPUnit_Framework_TestCase
```

Comment:
you're testing the Base class here, not your specialized class. I think this test class is useless. You can remove it.